TINY-8939: Add allow attribute as a valid attribute for the iframe element in the Schema (#8073)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- New `text_patterns_lookup` option to provide additional text patterns dynamically #TINY-8778
- New promotion element has been added to the UI and can be disabled using the new `promotion` option #TINY-8840
- New `format_noneditable_selector` option to specify the `contenteditable="false"` elements that can be wrapped in a format. #TINY-8905

> - `allow` attribute is now a valid attribute od the `iframe` schema

Reviewer: ```suggestion
- `allow` is now a valid attribute for the `iframe` schema. #TINY-8939
```

> - `allow` is now a valid attribute for the `iframe` schema. #TINY-8939

Reviewer 2: Maybe. Having a back ticks item as the first thing looks a bit weird.

```suggestion
- Added `allow` as valid attribute for the `iframe` element in the editor schema. #TINY-8939
```
